natgeo/mortar
# [SYTO-1219] Coverage to mortar

Ticket: https://jira.natgeo.com/browse/SYTO-1219

- Added coverage
- Moved some components to class components to avoid conflicts with enzyme
- Some fixes in functions
- Fixed tests with no JSON config being passed (Pestle related).
- Upgrading Node to v6

To test it:
- `npm install`
- `gulp karma`
- open generated `coverage/index.html` and check it reflects tests coverage
- have a cocktail 🍸
